site/src/main/tut/guide.md

```diff
@@ -159,6 +159,30 @@ for (i <- Range(1, 10)) myTimedSleepyTask.unsafeRun
 implicitly[Registry].outputText
 ```
 
+## Using with Play 2.4.x
+
+It is possible to add instrumentation to your 2.4.x Play app Controllers very easily.
+A [Play filter](https://www.playframework.com/documentation/2.4.x/ScalaHttpFilters) named `PrometheusFilter` is available in the `play` contrib module and will add a Histogram metric named `http_request_duration_seconds` with the following labels:
+* method (the http method, i.e. GET / PUT)
+* path (the route path)
+* status (the http response code family, i.e. 2xx, 4xx, 5xx)
+
+Create a `Filter` class in the root of your `Play` service with the following implementation:
+
+```scala
+import com.google.inject.Inject
+import org.lyranthe.prometheus.client.play.filters.PrometheusFilter
+import play.api.http.HttpFilters
+
+class Filters @Inject()(prometheusFilter: PrometheusFilter) extends HttpFilters {
```

Comment:
Yeah - if you can check if it works via config file, that would be perfect. People will still need to create a Registry though. (I don't actually know how to put one into the Guide context btw)